test/fixtures/SpokePool.Fixture.ts

```diff
@@ -410,6 +431,40 @@ export async function modifyRelayHelper(
   };
 }
 
+export async function getUpdatedUSSDepositSignature(
+  depositor: SignerWithAddress,
+  depositId: number,
+  originChainId: number,
+  updatedOutputAmount: BigNumber,
+  updatedRecipient: string,
+  updatedMessage: string
+): Promise<string> {
+  const typedData = {
+    types: {
+      UpdateDepositDetails: [
+        { name: "depositId", type: "uint32" },
+        { name: "originChainId", type: "uint256" },
+        { name: "updatedOutputAmount", type: "uint256" },
+        { name: "updatedRecipient", type: "address" },
+        { name: "updatedMessage", type: "bytes" },
+      ],
+    },
+    domain: {
+      name: "ACROSS-V2",
+      version: "1.0.0",
+      chainId: Number(originChainId),
```

Comment:
Minor: originChainId should already be a number.

```suggestion
      chainId: originChainId,
```